backend/internal/models/media.go

```diff
@@ -0,0 +1,58 @@
+package models
+
+import "time"
+
+type MediaType string
+
+const (
+	TrackMedia MediaType = "track"
```

Comment:
Changed the name because Track and Album are now classes that implement the interface Media, and I thought switcihng the name would be less confusing.